### PR TITLE
function for more endpoints added

### PR DESCRIPTION
**Pull request by Scrounger**
- getting all system infos from '/ocs/v2.php/apps/serverinfo/api/v1/info?format=json'
- getting notifications from '/ocs/v2.php/apps/notifications/api/v2/notifications?format=json'
- getting apps from '/ocs/v1.php/cloud/apps?format=json'
- getting appInfos from '/ocs/v1.php/cloud/apps/${appName}?format=json'

### CODE REVIEW

**Comment by hobigo** [comment]
Hi Scrounger,
I found your contribution today. 
Changes are looking good.
I moved some older uncommited changes to the repo.
Please give me some time check
Thanks
Holger

**Comment by Scrounger** [comment]
@hobigo 

I resolved the conflict. But i saw that you have some hard coded stuff here:
https://github.com/hobigo/nextcloud-node-client/blob/b486dd61e2634b628d6ce220073ff14de8b8f498/src/client.ts#L1142

And a german email adress. Do you speak german? :-)

File: src/client.ts [hunk 1]
```diff
@@ -52,21 +52,21 @@ interface IStat {
...
 
 export interface ISysInfoNextcloudClient {
     "version": string;
 }
 
 export interface ISysInfoNextcloud {
-    "system": ISysInfoNextcloudSystem;
+    "system": object;
+    "storage": object;
+    "shares": object;
```
Comment (hobigo) [inline src/client.ts:63]: OK, will change object to a more specific type later

**Comment by hobigo** [comment]
Great work!
You can reach me using the email you found also in german
Thanks Holger

**Pull request merged**